src/core/fem/src/discretization/4C_fem_discretization_utils.cpp

```diff
@@ -121,4 +121,35 @@ void Core::FE::Utils::do_initial_field(const Core::Utils::FunctionManager& funct
   }
 }
 
+/*----------------------------------------------------------------------------*
+ *----------------------------------------------------------------------------*/
+Core::LinAlg::MultiVector<double> Core::FE::Utils::average_element_to_nodal_values(
+    const Discretization& discretization,
+    const Core::LinAlg::MultiVector<double>& element_multi_vector)
```

Comment:
Data is const. And the type can be seen from the declaration and must not be part of the argument's name.
```suggestion
    const Core::LinAlg::MultiVector<const double>& element_data)
```